pilot/pkg/security/authn/utils/utils_test.go

```diff
@@ -115,11 +119,39 @@ func TestBuildInboundFilterChain(t *testing.T) {
 				node: &model.Proxy{
 					Metadata: &model.NodeMetadata{},
 				},
+				listenerProtocol: networking.ListenerProtocolTCP,
 			},
 			// Two filter chains, one for mtls traffic within the mesh, one for plain text traffic.
 			want: []networking.FilterChain{
 				{
-					TLSContext: tlsContext,
+					TLSContext: &auth.DownstreamTlsContext{
```

Comment:
This seems a direct duplicate of the tlsContext defined in the beginning of the function, could you deduce the duplicate by putting them in an anonymous function here, it could also be used in below.

Reply:
Thanks Fixed